src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -405,14 +467,13 @@ def create(dataset, target, feature, max_iterations=10,
 
         # Calculate training metric
         for data, label in train_data:
-            outputs = custom_NN.predict(data)
-            train_metric.update(label, outputs)
+            result = custom_NN.evaluate(data, label)
+            train_metric.update(result['accuracy'])
         train_data.reset()
 
-        # Calculate validation metric
         for data, label in validation_data:
-            outputs = custom_NN.predict(data)
-            validation_metric.update(label, outputs)
+            result = custom_NN.evaluate(data, label)
+            validation_metric.update(result['accuracy'])
```

Comment:
Could we stick with the old interface for the accuracy metric, which was chosen to avoid the kind of error you're introducing here?

Reply:
Done